server/src/controllers/mealControllers.js

```diff
@@ -1,86 +1,115 @@
-import meals from "../db/meals";
+import fs from "fs";
+import Meal from "../models/meals";
 
 class MealControllers {
-  //Get Meal for the Single day
-  static getMeal(req, res) {
-    return res.status(200).json({
-      status: res.statusCode,
-      data: meals
-    });
+  static async getMeal(req, res) {
+    try {
+      const meals = await Meal.findAll({
+        where: { catererId: 2 }
+      });
+      return res.status(200).json({
+        status: "success",
+        message: "Meals Retrieved",
+        data: meals
+      });
+    } catch (err) {
+      return res.status(500).json({
+        status: "error",
+        message: "Failed to Retrieve Meals"
+      });
+    }
   }
 
-  // Post/add a Meal Option
-  static postMeal(req, res) {
-    if (!req.body.name) {
-      return res.status(400).send({
-        status: res.statusCode,
-        message: "Meal name is required"
+  static async postMeal(req, res) {
+    try {
+      const { name, price } = req.body;
+      const { image } = req.files;
+      const imageUrl = `/src/images/${image.name}`;
+      const meal = await Meal.create({
+        name,
+        price,
+        imageUrl,
+        catererId: 2
```

Comment:
Why are you manually adding the catererId? This is a very dangerous idea as all meals will now have an id of 2